source/Nevermore.IntegrationTests/IntegrationTestDatabase.cs

```diff
@@ -16,7 +16,7 @@ namespace Nevermore.IntegrationTests
     public class IntegrationTestDatabase
     {
         readonly TextWriter output = Console.Out;
-        readonly string SqlInstance = "(local)\\SQLEXPRESS,1433";
+        readonly string SqlInstance = Environment.GetEnvironmentVariable("NevermoreTestServer") ?? "(local)\\SQLEXPRESS,1433";
```

Comment:
I'm curious about this addition. Was it you have a default instance and it didn't get picked up by .NET Core? If so we have the `SqlAlias` library for that.

Reply:
The /SQLEXPRESS alias redirection is esoteric. I have been ensuring that database connection dependencies are configurable through normal app settings or environment variables everywhere.